pkg/crc/machine/status.go

```diff
@@ -73,8 +67,13 @@ func (client *client) Status() (*types.ClusterStatusResult, error) {
 	}, nil
 }
 
-func (client *client) getDiskDetails(sshRunner *crcssh.Runner) (int64, int64) {
+func (client *client) getDiskDetails(ip string, bundle *bundle.CrcBundleInfo) (int64, int64) {
 	disk, err, _ := client.diskDetails.Memoize("disks", func() (interface{}, error) {
+		sshRunner, err := crcssh.CreateRunner(ip, getSSHPort(client.useVSock()), constants.GetPrivateKeyPath(), constants.GetRsaPrivateKeyPath(), bundle.GetSSHKeyPath())
+		if err != nil {
```

Comment:
While looking at this code, without looking at more code, I'm not sure what the benefit is. I assume `CreateRunner` creates the TCP connection to the VM, so moving it here and running it only when needed saves some time?
I think it would be fine to squash it in the first commit.

Reply:
Yes I squashed this.
The connection is created only when used, so it was just esthetic.